package.json

```diff
@@ -94,6 +95,7 @@
         "@tanstack/react-virtual": "^3.0.0-beta.39",
         "@types/css": "^0.0.33",
         "@types/memoizee": "^0.4.8",
+        "@types/mustache": "^4.2.6",
```

Comment:
I'm sorry, It might come out as nitpicks but it's part of the rules of contributing to OSS to only changes lines that are relevent to your PR.